src/_provider/AniList/metaOverview.ts

```diff
@@ -468,7 +484,7 @@ export class MetaOverview extends MetaOverviewAbstract {
       reviews.push({
         body: {
           people: i.rating,
-          date: timestampToShortDate(i.createdAt * 1000),
+          date: getDateInLocale(i.createdAt * 1000, 'short'),
```

Comment:
Wrong in this case. Look at https://github.com/MALSync/MALSync/pull/2698/files?diff=split&w=1#diff-38e47b4eeea96b1634986e199da0917411a0e085aeab0a6949875cd6e7b7bee7R246

Reply:
My bad, forgot there should be "relative" date and not "static"